src/Tests/Cask.Tests/CaskSecretsTests.cs

```diff
@@ -12,10 +10,11 @@
 using static CommonAnnotatedSecurityKeys.Limits;
 
 namespace CommonAnnotatedSecurityKeys.Tests;
-
-[ExcludeFromCodeCoverage]
 public abstract class CaskTestsBase
 {
+    private protected static new HashSet<char> s_validBase64Url =
```

Comment:
```suggestion
    private protected static readonly HashSet<char> s_validBase64Url =
```

Build is failing on the `new` and should be `readonly`.